app/components/post_list/index.tsx

```diff
@@ -235,6 +235,7 @@ const PostList = ({
                     <ThreadOverview
```

Comment:
I really don't know if there is a difference or if is even possible as the statements return booleans while the swtich would be a string, also it will need to evaluate the result of each function regardless.. maybe is something that idk or not familiar with, but I don't see how we can use a switch here